File: CSharpMain/Average.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Indicators
{
    public class Average
    {
        private double _sum = 0;
        private long _count = 0;
        public double Value => _count == 0 ? 0 : _sum / _count;

        public void Add(double value)
        {
            _sum += value;
            _count++;
        }
    }
}

File: PyMain/average.py
class average:
    def __init__(self) -> None:
        self.sum = 0.0
        self.count = 0

    def add(self, value: float) -> None:
        self.sum += value
        self.count += 1

    @property
    def value(self) -> float:
        return 0.0 if self.count == 0 else self.sum / self.count
